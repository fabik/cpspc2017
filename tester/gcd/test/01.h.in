7
13 20 16 4 29 26 5
