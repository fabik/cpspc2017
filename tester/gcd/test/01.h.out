1
30
56
76
92
105
113
